galsim/wfirst/wfirst_detectors.py

```diff
@@ -138,24 +189,22 @@ def allDetectorEffects(img, prev_exposures=(), rng=None, exptime=default_exptime
     poisson_noise = galsim.PoissonNoise(rng)
     img.addNoise(poisson_noise)
 
-    # Quantize: have an integer number of photons in every pixel after inclusion of sky noise.
-    img.quantize()
-
     # Reciprocity failure (use WFIRST routine, with the supplied exposure time).
     addReciprocityFailure(img, exptime=exptime)
 
+    # Quantize.
+    img.quantize()
+
```

Comment:
I think this is wrong.  Didn't we decide that it goes before reciprocity failure, and that if it's there, the poisson noise already makes integers, so we don't need to do anything else?

Reply:
c.f. discussion in https://github.com/GalSim-developers/GalSim/pull/1022/files

Reply:
It's moved to the front of the reciprocity failure.